lib/cosmos/tools/cmd_sender/cmd_sender.rb

```diff
@@ -690,6 +690,10 @@ def update_cmd_params(ignored_toggle = nil)
                 end
                 if !value_text.is_printable?
                   value_text = "0x" + value_text.simple_formatted
+                # Add quotes around string defaults which are stings so CmdSender interprets them correctly
```

Comment:
You misspelled strings and also this code doesn't work.   The STRING default is always a string even when it is initally input as a hex value (the hex value gets properly converted to a string).

Reply:
This works because the value-text is NOT printable so the hex string number case uses the first part of the if and gets the '0x' added. This is supposed to handle the case where the default isn't a hex value. Note that the only cases we support is that it's either a hex string default or else it's treated as a string with or without quotes. Thus you can't create a default of "\x01\x02" for example in a 2 byte block because it will get interpreted as a 8 character string. That's probably another ticket though ...